velox/docs/functions/spark/math.rst

```diff
@@ -54,6 +54,10 @@ Mathematical Functions
     Returns ``x`` rounded down to the nearest integer.
     Supported types are: BIGINT and DOUBLE.
 
+.. function:: log1p(x) -> double
+
+    Returns the natural logarithm of the “given value ``x`` plus one”.
```

Comment:
Spark docs say

`If expr is less than or equal to -1 the result is NULL.`

Let's add this and change implementation to match.

Reply:
Done